src/utils/style.ts

```diff
@@ -59,13 +54,11 @@ export const cssFontSizeToken = ({
     }
   | {
       type: Extract<TextType, 'tag'>;
-      size: NoteFontSize;
+      size: TagFontSize;
```

Comment:
The designation was incorrect (the values were the same, so it did not sound like a problem).